packages/wallet-connect-sdk-core/src/index.ts

```diff
@@ -199,36 +499,62 @@ export class WcSdk {
         })
     }
 
-    static async getSession(wcClient: Client) {
+    /**
+     * Gets any existing connection, it should be called after the initialization, to retrieve the session of connections made previously
+     */
+    static async getSession (wcClient: Client): Promise<SessionTypes.Settled | null> {
         if (wcClient?.session.topics.length) {
             return await wcClient.session.get(wcClient.session.topics[0])
         } else {
             return null
         }
     }
 
-    static getAccountInfo(session: SessionTypes.Settled, accountIndex?: number) {
+    /**
+     * gets the information of a connected account
+     * @param session connection session
+     * @param accountIndex index of one of the connected accounts
+     * @return an array that represents the address and chain
+     */
+    static getAccountInfo (session: SessionTypes.Settled, accountIndex?: number): string[] | null {
         const index = accountIndex ?? 0
         if (session.state.accounts.length <= index) {
             return null
         }
         return session.state.accounts[index].split(':')
     }
 
-    static getAccountAddress(session: SessionTypes.Settled, accountIndex?: number) {
+    /**
+     * gets the address of the connected account
+     * @param session connection session
+     * @param accountIndex the index of the account to retrieve, gets the first account if no index is provided
+     * @return the address of the connected account of the wallet

```

Comment:
```suggestion
     * @return the address of the connected account

```
same suggestion again. Check if previous was accepted.